<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lifetimeMeasurement</class>
 <widget class="QWidget" name="lifetimeMeasurement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lifetimeMeasurement</string>
  </property>
  <widget class="QComboBox" name="comboBox_laserController">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_laserController">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>eg: M00460823</string>
   </property>
   <property name="text">
    <string>Laser Controller:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_oscilloscope">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_oscilloscope">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>eg: DS6D150100004</string>
   </property>
   <property name="text">
    <string>Oscilloscope:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_refresh">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>83</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="PlotWidget" name="widget_currentDecay" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="PlotWidget" name="widget_averagedDecay" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_folder">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_file">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_dataSavedAt">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data saved at:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_folder">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>folder:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filename">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>filename:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectFolder">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select folder</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_acquiringTime">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>270</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_acquiringTime">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data acquiring time (s):</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_timeBetweenTurns">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>270</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_timeBetweenTurns">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>A resting time prevents too frequent switching of the laser diode controller</string>
   </property>
   <property name="text">
    <string>Resting time between turns (s) :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>580</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Decay of current turn</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>580</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Decay of averaged turns</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_turns">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number  of turns:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_turns">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
